Resources/Prototypes/Catalog/VendingMachines/Inventories/engivend.yml

```diff
@@ -5,7 +5,9 @@
     Multitool: 10
     ClothingEyesGlassesMeson: 10
     ClothingHeadHatWelding: 6
+    HolofanProjector: 10    # Frontier
```

Comment:
Holofan is a bit much, I think we should still restrict them.

Reply:
They exists on ships plenty as of now, also not much abused as they main use is redoing ship walls for building.